packages/repco-frontend/app/components/primitives/input.tsx

```diff
@@ -1,51 +1,113 @@
+import * as Separator from '@radix-ui/react-separator'
+import * as Tooltip from '@radix-ui/react-tooltip'
+import React from 'react'
 import type { VariantProps } from 'class-variance-authority'
 import { cva, cx } from 'class-variance-authority'
 import type { ClassProp } from 'class-variance-authority/dist/types'
 
 const styles = cva(
-  'border transition-colors duration-100 cursor-default disabled:opacity-50',
+  'border rounded-lg transition-colors duration-100 outline-none focus:border-primary-500 disabled:opacity-50',
   {
     variants: {
       disabled: {
-        true: 'opacity-70 pointer-events-none cursor-not-allowed',
```

Comment:
why remove "cursor-not-allowed"?  it is literally was "disabled" mean

Reply:
Fixed it. I probably lost it when I rewrote the file...